.github/workflows/license_review.yml

```diff
@@ -0,0 +1,21 @@
+name: license_review
+on:
+  pull_request:
```

Comment:
thought that perhaps we'd only want to do this check `on pull_request` since deps would be approved before merging to master, but I can add `on: push` too.

Reply:
during standup, we said just `on pull request` is fine.